[debug] [2022-08-26T10:09:33.654Z] ----------------------------------------------------------------------
[debug] [2022-08-26T10:09:33.657Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Shree\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2022-08-26T10:09:33.658Z] CLI Version:   11.8.0
[debug] [2022-08-26T10:09:33.658Z] Platform:      win32
[debug] [2022-08-26T10:09:33.659Z] Node Version:  v16.17.0
[debug] [2022-08-26T10:09:33.660Z] Time:          Fri Aug 26 2022 15:39:33 GMT+0530 (India Standard Time)
[debug] [2022-08-26T10:09:33.661Z] ----------------------------------------------------------------------
[debug] 
[debug] [2022-08-26T10:09:33.780Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2022-08-26T10:09:33.781Z] > authorizing via signed-in user (nagdatt.h.gajjam@gmail.com)
[debug] [2022-08-26T10:09:33.781Z] [iam] checking project supplychain-c2304 for permissions ["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get","firebasehosting.sites.update"]
[debug] [2022-08-26T10:09:33.785Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/supplychain-c2304:testIamPermissions [none]
[debug] [2022-08-26T10:09:33.785Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/supplychain-c2304:testIamPermissions x-goog-quota-user=projects/supplychain-c2304
[debug] [2022-08-26T10:09:33.786Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/supplychain-c2304:testIamPermissions {"permissions":["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2022-08-26T10:09:35.808Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/supplychain-c2304:testIamPermissions 200
[debug] [2022-08-26T10:09:35.808Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/supplychain-c2304:testIamPermissions {"permissions":["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2022-08-26T10:09:35.809Z] >>> [apiv2][query] POST https://iam.googleapis.com/v1/projects/supplychain-c2304/serviceAccounts/supplychain-c2304@appspot.gserviceaccount.com:testIamPermissions [none]
[debug] [2022-08-26T10:09:35.810Z] >>> [apiv2][body] POST https://iam.googleapis.com/v1/projects/supplychain-c2304/serviceAccounts/supplychain-c2304@appspot.gserviceaccount.com:testIamPermissions {"permissions":["iam.serviceAccounts.actAs"]}
[debug] [2022-08-26T10:09:37.600Z] <<< [apiv2][status] POST https://iam.googleapis.com/v1/projects/supplychain-c2304/serviceAccounts/supplychain-c2304@appspot.gserviceaccount.com:testIamPermissions 404
[debug] [2022-08-26T10:09:37.600Z] <<< [apiv2][body] POST https://iam.googleapis.com/v1/projects/supplychain-c2304/serviceAccounts/supplychain-c2304@appspot.gserviceaccount.com:testIamPermissions {"error":{"code":404,"message":"Unknown service account","status":"NOT_FOUND"}}
[debug] [2022-08-26T10:09:37.602Z] [functions] service account IAM check errored, deploy may fail:HTTP Error: 404, Unknown service account {"name":"FirebaseError","children":[],"context":{"body":{"error":{"code":404,"message":"Unknown service account","status":"NOT_FOUND"}},"response":{"statusCode":404}},"exit":1,"message":"HTTP Error: 404, Unknown service account","status":404}
[debug] [2022-08-26T10:09:37.604Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/supplychain-c2304 [none]
[debug] [2022-08-26T10:09:38.542Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/supplychain-c2304 200
[debug] [2022-08-26T10:09:38.542Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/supplychain-c2304 {"projectId":"supplychain-c2304","projectNumber":"341447576190","displayName":"supplyChain","name":"projects/supplychain-c2304","resources":{"hostingSite":"supplychain-c2304"},"state":"ACTIVE","etag":"1_7fbba9b5-541b-4906-ad41-47cf29184bc8"}
[info] 
[info] === Deploying to 'supplychain-c2304'...
[info] 
[info] i  deploying firestore, functions, hosting 
[debug] [2022-08-26T10:09:38.910Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:387:5)
    at validateString (node:internal/validators:121:11)
    at Object.join (node:path:429:7)
    at Config.path (C:\Users\Shree\AppData\Roaming\npm\node_modules\firebase-tools\lib\config.js:123:45)
    at getChildEnvironment (C:\Users\Shree\AppData\Roaming\npm\node_modules\firebase-tools\lib\deploy\lifecycleHooks.js:51:49)
    at runTargetCommands (C:\Users\Shree\AppData\Roaming\npm\node_modules\firebase-tools\lib\deploy\lifecycleHooks.js:72:14)
    at C:\Users\Shree\AppData\Roaming\npm\node_modules\firebase-tools\lib\deploy\lifecycleHooks.js:124:24
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async chain (C:\Users\Shree\AppData\Roaming\npm\node_modules\firebase-tools\lib\deploy\index.js:33:9)
    at async deploy (C:\Users\Shree\AppData\Roaming\npm\node_modules\firebase-tools\lib\deploy\index.js:67:5)
[error] 
[error] Error: An unexpected error has occurred.
